internal/engine/image_build_and_deployer.go

```diff
@@ -168,9 +168,14 @@ func (ibd *ImageBuildAndDeployer) push(ctx context.Context, ref reference.NamedT
 	ps.StartPipelineStep(ctx, "Pushing %s", ref.String())
 	defer ps.EndPipelineStep(ctx)
 
+	cbSkip := false
+	if iTarget.IsCustomBuild() {
+		cbSkip = iTarget.CustomBuildInfo().DisablePush
+	}
+
```

Comment:
tbh you could just to `cbSkip := iTarget.CustomBuildInfo().DisablePush` (or even take all this out and in the conditional below, just put `iTarget.CustomBuildInfo().DisablePush`)

Reply:
Couldn't it be nil though?

Reply:
Nah, if it's not a custom build, this func returns an EMPTY `CustomBuild` (not a nil one). though i guess you're right that that's maybe not functionality we want to rely on...?